blockchain/projects.go

```diff
@@ -56,7 +56,7 @@ type Projects struct {
 }
 
 // Reset the blockchain state and return the new account models
-func (p *Projects) Reset(projectID uuid.UUID, em *blockchain) ([]*model.Account, error) {
+func (p *Projects) Reset(projectID uuid.UUID) ([]*model.Account, error) {
```

Comment:
Is this still used anywhere on the API? if not I would advise removing it.

Reply:
No it's not. It was added in the file refactor specifically for this.

Reply:
Then we could remove it no?